pkg/gatherers/clusterconfig/gather_lokistack.go

```diff
@@ -0,0 +1,137 @@
+package clusterconfig
+
+// nolint: dupl
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/klog/v2"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+const LokiStackResourceLimit = 20
+
+// GatherOpenshiftLogging Collects `clusterlogging.logging.openshift.io` resources.
```

Comment:
It would be nice to mention the limitation related to namespaces and the number of resources.

Reply:
Where would be the best place to mention it?

Reply:
Here in the comments. The comment is used as an input for generating the `gathered-data.md` file

Reply:
Done